src/applications/new-28-1900/tests/e2e/save-in-progress.cypress.spec.jsx

```diff
@@ -0,0 +1,40 @@
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../../config/form';
+import manifest from '../../manifest.json';
+import userSip from '../fixtures/mocks/user-sip.json';
+import submit from '../fixtures/mocks/submit.json';
+import featureToggles from '../fixtures/mocks/featureToggles.json';
+import minimalFlow from '../fixtures/data/minimalFlow.json';
+
+// This test ensures that the save-in-progress functionality of the 28-1900 form works as expected.
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataDir: null,
+    dataSets: [{ title: 'minimalFlow', data: minimalFlow }],
+
+    pageHooks: {
+      introduction: ({ afterHook }) => {
+        afterHook(() => {
+          cy.contains('button', 'Continue your application').click();
```

Comment:
This line ensures that the continue button exists, rather than the default start button.